src/main/java/de/internetsicherheit/brl/bloxberg/cache/BlockDataExtractor.java

```diff
@@ -18,49 +16,42 @@
 public class BlockDataExtractor {
 
     private BloxbergClient client;
-    private final String OUTPUTDIRECTORYNAME = System.getProperty("user.dir") + "/output/";
-    private File outputdirectory;
     private File outputfile;
     private int start;
     private int stop;
     private String filename;
 
     public BlockDataExtractor(String[] args) throws IOException {
-
         this.client = new BloxbergClient(args[0]);
-        outputdirectory = new File(OUTPUTDIRECTORYNAME);
+        String OUTPUTDIRECTORYNAME = System.getProperty("user.dir") + "/output/";
```

Comment:
Only constants are written in all caps. This variable was definitely better and more idiomatic as a constant like it was in the old version of the code. 

Maybe just changing to `OUTPUT_DIRECTORY_NAME`.